src/OrchardCore.Modules/OrchardCore.Seo/Indexes/SeoMetaPartContentIndexHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading.Tasks;
+using OrchardCore.ContentManagement;
+using OrchardCore.Indexing;
+using OrchardCore.Seo.Models;
+
+namespace OrchardCore.Seo.Indexes;
+
+public class SeoMetaPartContentIndexHandler : IContentItemIndexHandler
```

Comment:
Ok, I don't really see when I will require to search in Lucene a Seo meta title. Though, can be useful if needed.

Reply:
I think it's fine to add it in the indices, might help for the full text search.